protocols/luci-proto-wireguard/root/usr/libexec/rpcd/luci.wireguard

```diff
@@ -15,6 +15,14 @@ local methods = {
 			return {keys = {priv = prv, pub = pub}}
 		end
 	},
+	getPublicKeyFromPrivate = {
+		args = {privkey = "privkey"},
+		call = function(args)
+			local pubkey = sys.exec("echo '" .. args.privkey .. "' | wg pubkey 2>/dev/null"):sub(1, -2)
```

Comment:
This is a shell injection with root privileges. You should use https://openwrt.github.io/luci/api/modules/luci.util.html#shellquote as it has been done in 44445a8097d05dbcc807c95c5b2c016f1a49a350

Reply:
Sure